src/utilities/__tests__/astFromValue-test.ts

```diff
@@ -96,6 +121,11 @@ describe('astFromValue', () => {
       value: '123',
     });
 
+    expect(astFromValue(9007199254740993n, GraphQLFloat)).to.deep.equal({
```

Comment:
The change in type of this test is because astFromValue is not safe, set to be replaced by `valueToLiteral()` for all remaining uses in https://github.com/graphql/graphql-js/pull/3814

But a bigint of sufficient size, -- as far as I know -- cannot be converted to a JS float.

We also in this PR do not properly handle GraphQLFloat scalar parsing and serialization of small BigInts, I will push a failing test to demonstrate what I mean.

We also probably should update the logic within the new `valueToLiteral()` utility. It's possible that overall we can just delay landing this PR until after #3814